src/useRedux.js

```diff
@@ -6,10 +6,10 @@ export const useRedux = (selectors, actionCreators) => {
   const { getState, dispatch, subscribe } = store;
   const [reduxState, setReduxState] = useState(getState());
 
-  useEffect(() => {
-    const unsubscribe = subscribe(() => setReduxState(getState()));
-    return () => unsubscribe();
-  }, []);
+  useEffect(
+    () => subscribe(() => setReduxState(getState())),
+    []
+  );
```

Comment:
It should be all in one line. Please run `yarn format`

Reply:
Done! Tests are re-running.